assets/js/pages/search.js

```diff
@@ -95,4 +113,12 @@
     }
   };
 
+  isInPackageSearch = function(query){
```

Comment:
Same here, declare them with `var` to make them local to the module scope's. You might need to move it up tho